wolfssl/internal.h

```diff
@@ -1140,6 +1153,10 @@ enum {
     TLS_SHA256_SHA256            = 0xB4,
     TLS_SHA384_SHA384            = 0xB5,
 
+    /* ARIA-GCM, first byte is 0xC0 (ECC_BYTE) */
```

Comment:
would be nice if this comment references the RFC https://www.rfc-editor.org/rfc/rfc6209.html#section-5